runtime/queries/nix/highlights.scm

```diff
@@ -33,6 +33,11 @@
 
 (uri) @string.special.uri
 
+; boolean
+((identifier) @constant.builtin.boolean (#match? @constant.builtin.boolean "^(true|false)$")) @constant.builtin.boolean
```

Comment:
Should this just be `["true" "false"] @constant.builtin.boolean`?

Reply:
I think this doesn't work as true and false is recognized as identifier:
`thread 'main' panicked at 'Query error at 37:3. Invalid node type true', helix-core/src/syntax.rs:207:29`